core/state/database.go

```diff
@@ -75,7 +75,7 @@ type Database interface {
 
 	Transitioned() bool
 
-	InitTransitionStatus(bool, bool)
+	InitTransitionStatus(bool, bool, *common.Address, *common.Hash, *bool)
```

Comment:
Note to self: this change is actually useless since we call all the `Set*` functions in `MakePreState`